File: _Trigonometric_Method_of_Defining_an_Ellipse.py

import numpy as np
import matplotlib.pyplot as plt

a = 5
b = 3

theta = np.linspace(0, 2*np.pi, 400)
x = a * np.cos(theta)
y = b * np.sin(theta)

plt.plot(x, y)
plt.gca().set_aspect('equal', adjustable='box')
plt.title("Trigonometric Method of Defining an Ellipse")
plt.xlabel("X-axis")
plt.ylabel("Y-axis")
plt.grid(True)
plt.show()

File: _Trigonometric_Method_of_Defining_an_Ellipse.m

% Trigonometric Method of Defining an Ellipse
a = 5;
b = 3;

theta = linspace(0, 2*pi, 400);
x = a * cos(theta);
y = b * sin(theta);

plot(x, y, 'r');
axis equal;
title('Trigonometric Method of Defining an Ellipse');
xlabel('X-axis');
ylabel('Y-axis');
grid on;
